README.md

```diff
@@ -93,6 +102,20 @@ Ansible Best Practices
   Avoid duplicating the remote full path in the role directory, however, as that creates unnecessary depth in
   the file tree for the role. Grouping sets of similar files into a subdirectory of `templates` is allowable,
   but avoid unnecessary depth to the hierarchy.
+* Example playbooks are to live in the root directory, not in a
+  separate subdirectory, and prefixed with `example-`.
```

Comment:
So in the exciting new age of collections, there is a pre-defined "playbooks" folder at the root of collections where example playbooks should live.

Might we want to revisit this recommendation?

Reply:
first of all, it has nothing to do with the subject of the PR (Python standards).

Reply:
concerning the item here, `playbooks` exist in collections, but not in roles. This recommendation is for example playbooks for each role. (Do you suggest to collect example playbooks for all the roles in a collection in the collection's `playbooks` directory?)

Reply:
That would be suggestion, yes. Obviously, if they're still being distributed as a bare role, then they can live where they are in accordance with this suggestion. But once a role moves into a collection, we should follow the collection's best practices.

Reply:
Where is this best practice for collections defined? https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html#playbooks-directory is not very informative. It does not say what is the playbook directory good for.

Reply:
It sure doesn't give us much information. But if there's a playbooks directory inside of a collection, then we should start putting our playbooks in there for now. Once more structure is defined within it by the rest of the community, we can begin giving further guidance here. But, for the time being, we should just default to putting any playbooks in that directory.

In the OASIS world, we don't have any playbooks that even qualify for this. So the lack of more detailed guidance from the Ansible community isn't really a big deal as it's all hypothetical.

Reply:
Well it does not say what playbooks (example playbooks rather than playbooks that are ready to use?) it also does not say whether those be playbooks (examples?) for anything in the collection or just playbooks that apply to the collection as a whole (e.g those that use more than one item in the collection).

This topic is relevant for Linux System Roles, as we have plenty of example playbooks already.